examples/ping.rs

```diff
@@ -89,7 +90,24 @@ async fn main() -> Result<(), Box<dyn Error>> {
     loop {
         match swarm.select_next_some().await {
             SwarmEvent::NewListenAddr { address, .. } => info!("Listening on {address:?}"),
-            SwarmEvent::Behaviour(event) => info!("{event:?}"),
+            SwarmEvent::Behaviour(event) => {
+                // Get the round-trip duration for the pings.
+                // This value is already captured in the BehaviourEvent::Ping's `Success::Ping`
+                // field.
+                info!("{event:?}");
+                match event {
+                    BehaviourEvent::Ping(ping_event) => {
+                        let result: Success = ping_event.result?;
+                        match result {
+                            Success::Ping { rtt } => {
+                                info!("Ping RTT: {rtt:?}");
+                            }
+                            Success::Pong => {}
+                        }
+                    }
+                    _ => {}
```

Comment:
we should log these cases too, good to see any errors that may arise (and the pong responses)